src/planscape/plan/models.py

```diff
@@ -71,6 +70,23 @@ class GeneratedProjectAreas(models.Model):
         null=True)
 
 
+class ScenarioSet(models.Model):
+    # TODO: Change "null=True" so that owner is not nullable. Currently owner can be null because
+    # we want alpha users to not be signed in.
+    owner = models.ForeignKey(
+        User, on_delete=models.CASCADE, null=True)  # type: ignore
+
+    project = models.ForeignKey(
+        Project, on_delete=models.CASCADE)  # type: ignore
+
+    # Scenario Parameters:
+
+    # TODO: Limit number of allowed priorities
+    priorities = models.ManyToManyField('conditions.Condition')
+
+    # TODO: Add constraints
```

Comment:
There are only 4 constraints (that I know of):
min-cost, max-cost, min-area, max-area - maybe they can be expressed as individual fields in the url?

I'm still unclear on thresholds (and whether they're applied to individual pixels, to project areas, etc).

Reply:
Noted thanks! I'll hold off on that decision for a bit.